README.md

```diff
@@ -232,6 +232,17 @@ meta:
     interval: 5 # Publishing interval in seconds, default value is 5
 ```
 
+To collect metrics via `Prometheus`, we configure metrics like this :
+
+```yaml
+metrics :
+  type : promu
+  interval : 60
```

Comment:
Interval is not needed for pull strategy

Reply:
You are right, even if we need to save periodically metrics in memory. Ok, I will use the default interval which 5.

Reply:
You are right, even if we need to save periodically metrics in memory. Ok, I will use the default interval which 5.